cubi_isa_templates/isatab-cytof/hooks/pre_gen_project.py

```diff
@@ -0,0 +1,7 @@
+import sys
+
+sample_factors = "{{cookiecutter.sample_factors}}".split(",")
+sample_factor_types = "{{cookiecutter.sample_factor_types}}".split(",")
+if not len(sample_factors) == len(sample_factor_types):
+    print("ERROR: Sample factors needs to be the same length as Sample factor types.")
+    sys.exit(1)
```

Comment:
Not sure if this is the best way to perform this check, especially in the SODAR context.